sandbox/settings.py

```diff
@@ -370,6 +371,54 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
         },
     }
 
+    # CKEditor available settings
+    CKEDITOR_SETTINGS = {
+        "language": "{{ language }}",
+        "skin": "moono-lisa",
+        "toolbarCanCollapse": False,
+        "contentsCss": "/static/css/ckeditor.css",
+        # Enabled showblocks as default behavior
+        "startupOutlineBlocks": True,
+        # Enable some plugins
+        # 'extraPlugins': 'codemirror',
+        # Disable element filter to enable full HTML5, also this will let
+        # append any code, even bad syntax and malicious code, so be careful
+        "removePlugins": "stylesheetparser",
+        "allowedContent": True,
+        # Image plugin options
+        "image_prefillDimensions": False,
+        # Justify text using shortand class names
+        "justifyClasses": ["text-left", "text-center", "text-right"],
+        # Default toolbar configurations for djangocms_text_ckeditor
+        "toolbar": "CMS",
+        "toolbar_CMS": [
+            ["Undo", "Redo"],
+            ["ShowBlocks"],
+            ["Format", "Styles"],
+            ["RemoveFormat"],
+            ["Maximize"],
+            "/",
+            ["Bold", "Italic", "Underline", "-", "Subscript", "Superscript"],
+            ["JustifyLeft", "JustifyCenter", "JustifyRight"],
+            ["Link", "Unlink"],
+            [
+                "Image",
+                "-",
+                "NumberedList",
+                "BulletedList",
+                "-",
+                "Table",
+                "-",
+                "CreateDiv",
+                "HorizontalRule",
+            ],
+            ["Source"],
+        ],
+    }
+    # Share the same config for djangocms_text_ckeditor field and derived
+    # CKEditor widgets/fields
+    CKEDITOR_SETTINGS["toolbar_HTMLField"] = CKEDITOR_SETTINGS["toolbar_CMS"]
+
```

Comment:
Let's not do like Open edX. I think we should have default settings declared directly in the plugin so that adding this plugin to a project just works.

I propose the following pattern: create a settings.py file in the plugin with somethinng like:
```
from django.conf import settings
CKEDITOR_SETTINGS = getattr(settings, 'CKEDITOR_SETTINGS', {...your whole dict definition...})
```
In the plugin, you can use the setting value by doing:
```
from myplugin.settings import CKEDITOR_SETTINGS

def foo():
    ...
    print(CKEDITOR_SETTINGS)

Reply:
As you may know, `CKEDITOR_SETTINGS` is common setting from djangocms_text_ckeditor itself that is used from its plugin. Furthermore we don't use it directly, this is the widget which get it from settings then pass it to the CKEditor UI configuration. Except rewriting our own widget, i don't think we can do what you are proposing.

The plugins with CKEditor UI can work when it's not setted, you may see this as a sample config from sandbox that we may eventually suggest from documentation.